src/main/client/java.client.ftl

```diff
@@ -346,7 +364,7 @@ public class FusionAuthClient {
       [#elseif param.type == "urlParameter"]
         .urlParameter("${param.parameterName}", ${(param.constant?? && param.constant)?then(param.value, param.name)})
       [#elseif param.type == "body"]
-        .bodyHandler(new JSONBodyHandler(${param.name}, objectMapper))
+        .bodyHandler(new JSONBodyHandler(${param.name}, objectMapper()))
```

Comment:
You need to make this same change on lines `385` and `386` for use in the `JSONResponseHandler`. 

I suppose you could optionally omit this - but then the `setObjectMapper` is really `setObjectMapperForRequestSerialization` only and will never affect the API response. 

Perhaps this was your intent? If so - we should discuss. It does solve your specific issue, but it seems to me that we should solve this more generically and not assume the only reason you may want to provide your own `ObjectMapper` is for serialization.

Reply:
I had really only thought about the serialization for our case but considering it there's probably more reason to want to customize jackson for the response if you are an external user of our client.